### PR TITLE
Fix staging issue for compat Java 8 projects

### LINKED ISSUE #52: Standard staging fails for Compat Java 8 projects without Dockerfile

### PR DESCRIPTION
**Pull request by meltsufin**
Fixes #52 
Move up to version 0.1.3-SNAPSHOT of the appengine-plugins-core library

### CODE REVIEW

File: src/main/java/com/google/cloud/tools/maven/StageMojo.java [hunk 1]
```diff
@@ -193,6 +196,11 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     if (new File(sourceDirectory.toString() + "/WEB-INF/appengine-web.xml").exists()) {
       getLog().info("Detected App Engine standard environment application.");
 
+      // force runtime to 'java' for compat projects using Java version >1.7
+      if (Float.parseFloat(getCompileTargetVersion()) > 1.7f) {
```
Comment (loosebazooka) [inline src/main/java/com/google/cloud/tools/maven/StageMojo.java]: do you have access to maven comparable version here?
https://maven.apache.org/ref/3.3.3/maven-artifact/apidocs/org/apache/maven/artifact/versioning/ComparableVersion.html

Comment (meltsufin) [inline src/main/java/com/google/cloud/tools/maven/StageMojo.java]: I do, but it seems more verbose. I don't think the Java target version can ever be anything but a real number.

Comment (loosebazooka) [inline src/main/java/com/google/cloud/tools/maven/StageMojo.java]: alright

**Comment by loosebazooka** [comment]
:rocket:

**Pull request merged**